projects/ngx-drag-scroll/src/lib/ngx-drag-scroll.component.ts

```diff
@@ -421,6 +436,15 @@ export class DragScrollComponent implements OnDestroy, AfterViewInit, OnChanges,
     this.refreshWrapperDimensions();
   }
 
+  private _setIsDragging(value: boolean) {
+    if (this._isDragging === value) {
+      return;
+    }
+
+    this._isDragging = value;
+    value ? this.dragStart.emit() : this.dragEnd.emit();
+  }
+
```

Comment:
How do you feel about using a private setter for `isDragging`. Is matches the getter/setter convension.
```suggestion
  private set isDragging(value: boolean) {
    if (this._isDragging === value) {
      return;
    }

     this._isDragging = value;
    value ? this.dragStart.emit() : this.dragEnd.emit();
  }
```

Reply:
Thanks for the tip. In older versions of typescript this did not work and I made a mistake out of habit.

Reply:
All good. Should the 'I' in `set IsDragging` be lowercase? Everything else looks good!

Reply:
Oh, can you also merge your PR with develop and resolve the conflicts please?

Reply:
Unfortunately, this does not work in this version of typescript (error TS2379: Getter and setter accessors do not agree in visibility).

Reply:
This feature has long been asked from Microsoft https://github.com/Microsoft/TypeScript/issues/2845 , but it never did.